.envrc

```diff
@@ -1,14 +1,3 @@
-#------------------------------------------------------------------------
-# Load Development Spack Environment (If Spack is installed.)
-#
-# Run 'direnv allow' from within the cloned repository to automatically
-# load the spack environment when you enter the directory.
-#------------------------------------------------------------------------
-if type spack &>/dev/null; then
-    . $SPACK_ROOT/share/spack/setup-env.sh
-    spack env activate -d .
-fi
-
```

Comment:
If you remove/don't have a spack.lock file does activating the environment actually mess with your install through pip? Or is this purely to speed up the direnv execution?

Reply:
addressed via https://github.com/spack/spack-gantry/pull/10/commits/f54656223688f4be216d69fea5493150f810c77b